digits/templates/models/images/classification/new.html

```diff
@@ -366,6 +366,11 @@ <h4 style="display:inline-block;">Python Layers</h4>
                     {{form.use_mean.tooltip}}
                     {{form.use_mean(class='form-control')}}
                 </div>
+                <div class="form-group{{mark_errors([form.mean_path])}}">
```

Comment:
Question to all reviewers: don't we need to support both server and client-side image files (like we do for Python layers)?